projects/atft/src/lib/controls/orbit-controls.directive.ts

```diff
@@ -83,18 +84,21 @@ export class OrbitControlsDirective implements AfterViewInit, OnChanges, OnDestr
     );
     this.controls.rotateSpeed = this.rotateSpeed;
     this.controls.zoomSpeed = this.zoomSpeed;
-    this.controls.addEventListener('change', this.childRenderers.first.render);
-    this.childRenderers.first.render();
+
+    this.requestRender = this.requestRender.bind(this);
+    this.controls.addEventListener('change', this.requestRender);
+    this.requestRender();
+  }
+
+  private requestRender() {
+    this.render.emit();
```

Comment:
Is the `this` here really `OrbitControlsDirective`?

Reply:
I assume that 
```
this.requestRender = this.requestRender.bind(this);
```
do magic and _this_ remains.
But I will rewrite to be more transparent